rbi/stdlib/etc.rbi

```diff
@@ -564,6 +564,10 @@ end
 class Etc::Group < Struct
   extend T::Generic
   Elem = type_member(:out, fixed: T.untyped)
+  class << self
+    extend T::Generic
+    Elem = type_member(fixed: T.untyped)
+  end
 end
```

Comment:
It turns out that `Etc::Group` both `extend`s and `include`s `Enumerable`, which is why we needed to modify this RBI.